src/actions/ActionHandlers.ts

```diff
@@ -0,0 +1,47 @@
+import { TasksState, NewTaskState } from 'src/reducers/types'
+import { Actions } from 'src/actions/types'
+import { Task } from 'src/types/global'
+
+const _ = require('lodash')
+
+export const getTasks = (state: TasksState, action: Actions) => {
+  return Object.assign({}, state, {
```

Comment:
Just a comment, you can use the spread `...` instead the `Object.assign`, just only a comment, you dont need to change this.